rel/i18n/emqx_bridge_snowflake_action_schema.hocon

```diff
@@ -0,0 +1,91 @@
+emqx_bridge_snowflake_action_schema {
+  snowflake.label:
+  """Upload to Snowflake"""
+  snowflake.desc:
+  """Action that takes incoming events and uploads them to the Snowflake."""
+
+  parameters.label:
+  """Snowflake action parameters"""
+  parameters.desc:
+  """Set of parameters for the action."""
+
+  aggreg_parameters.label:
+  """Snowflake Aggregated Mode action parameters"""
+  aggreg_parameters.desc:
+  """Set of parameters for the action in aggregated mode."""
+
+  aggregated_mode.label:
+  """Aggregated Snowflake Upload"""
+  aggregated_mode.desc:
+  """Enables time-based aggregation of incoming events and uploading them to the Snowflake as a single object."""
+
+  aggregation.label:
+  """Aggregation parameters"""
+  aggregation.desc:
+  """Set of parameters governing the aggregation process."""
+
+  aggregation_interval.label:
+  """Time interval"""
+  aggregation_interval.desc:
+  """Amount of time events will be aggregated in a single file on each node before uploading."""
```

Comment:
i don't understand: if it's a single file on each node, why should there be a pool of odbc connections and a pool of http clients.

Reply:
there could be multiple deliveries going on at the same time.  one batch might fire, then, while it's still going on, another batch fills and fires concurrently.